sound/soc/intel/boards/sof_sdw.c

```diff
@@ -1026,8 +1026,17 @@ static struct sof_sdw_codec_info codec_info_list[] = {
 				.dai_type = SOF_SDW_DAI_TYPE_JACK,
 				.dailink = {SDW_UNUSED_DAI_ID, SDW_JACK_IN_DAI_ID},
 			},
+			{
+				.direction = {true, false},
+				.dai_name = "cs42l43-dp6",
+				.dai_type = SOF_SDW_DAI_TYPE_AMP,
+				.dailink = {SDW_AMP_OUT_DAI_ID, SDW_UNUSED_DAI_ID},
+				.init = sof_sdw_cs42l43_spk_init,
+				.rtd_init = cs42l43_spk_rtd_init,
+				.quirk = SOF_CODEC_SPKR,
```

Comment:
I don't see in this PR any 'system-specific' quirk. Is this because for now the speakers are only enabled with a kernel parameter for e.g. debug? Or are we missing a SKU definition where that quirk needs to be set.

Reply:
Yeah I will also put this in the redone commit message currently we don't actually have any systems requiring this yet. But there are systems in discussion and it's a good stepping stone to the more complex bridge setups.